src/test/java/seedu/address/testutil/TypicalGroups.java

```diff
@@ -11,7 +11,9 @@
 
 
     public static final List<SuperGroup> getSuperGroups() {
-        ORBITAL.addSubGroup(TypicalSubGroups.ORBITAL_GROUP1);
+        if (ORBITAL.findSubGroup("Group1") == null) {
+            ORBITAL.addSubGroup(TypicalSubGroups.ORBITAL_GROUP1);
+        }
```

Comment:
Same as above, maybe abstract into a method into the UniqueList?

Reply:
I will add that to my todos too. I am lazy to change now hahahaha
I will do it along with command testcases.